insights/parsers/nginx_log.py

```diff
@@ -9,7 +9,8 @@
     for more usage information.
 """
 
-from .. import LogFileOutput, parser
+from insights import ContainerParser
+from insights import LogFileOutput, parser
```

Comment:
Could you change these to full path imports please?

```
from insights.core import ContainerParser, LogFileOutput
from insights.core.plugins import parser
```

Reply:
@ryan-blakley - Updated, please check

Reply:
Thank you.